### ISSUE TITLE
Aliased crates aren't understood

### ISSUE THREAD

**Issue by CleanCut** [intellij-rust/intellij-rust#2612]
If I alias a crate, intellij-rust doesn't seem to know anything about it anymore.  😭 This may be related to (or a duplicate of) #1658.

```rust
use my_crate_with_a_really_long_name as shortname;
use shortname::Thing;

...
    Thing::new();  // intellij-rust is like, "Unresolved reference: `Thing`
```

Not sure if this is a feature request, a bug, or both. 😉 Loving the plugin, by the way!  I've been bouncing between Vim, Atom, and IntelliJ for my Rust coding.  intellij-rust is keeping me on IntelliJ for the moment, though I prefer lighter-weight IDEs or editors in general.

**Comment by Undin** [comment]
@CleanCut looks like that it's duplicate of #2568 but not #1658

**Comment by Undin** [comment]
Fixes in #2685

**Comment by CleanCut** [comment]
You guys are so awesome!
